cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -147,6 +147,9 @@ func SetDefaults_NodeConfiguration(obj *NodeConfiguration) {
 	if len(obj.DiscoveryToken) == 0 && len(obj.DiscoveryFile) == 0 {
 		obj.DiscoveryToken = obj.Token
 	}
+	if obj.CRISocket == "" {
+		obj.CRISocket = DefaultCRISocket
```

Comment:
you need to remove this since `DefaultCRISocket` is no longer defined

Reply:
I remember it was defined when I review the code last time. `obj.CRISocket` should have a default value I think.

Reply:
I missed adding this since I did the initial work within the same branch as my first PR. I added the DefaultCRISocket to resolve this issue.